src/styles/index.ts

```diff
@@ -3008,6 +3008,13 @@ const styles = (theme: ThemeColors) =>
             ...spacing.mv3,
         },
 
+        sectionDividerLine: {
```

Comment:
Are we sure we need new styles for this? Can we check and see if we have an existing divider style or component we can just reuse?

Reply:
No we don't have. We need a different vertical margin and remove the `flexGrow` property.

Reply:
Got it.